tests/gql_behave/tests/memgraph_V1/features/match.feature

```diff
@@ -469,6 +469,26 @@ Feature: Match
             | P12DT3M  |
             | P13DT12H |
 
+    Scenario: Test match with order by and datetime
+        Given an empty graph
+        And having executed:
+            """
+            CREATE({a: DATETIME('2024-01-22T08:11:31[Etc/UTC]')}),
+                  ({a: DATETIME('2024-01-22T08:12:31[Etc/UTC]')}),
+                  ({a: DATETIME('2024-01-22T09:12:31[Europe/Zurich]')}),
+                  ({a: DATETIME('2024-01-22T09:12:31[Europe/Warsaw]')})
```

Comment:
Can we add edge cases like +18, -18 hours, not 00 minutes but [30, 45](https://www.timeanddate.com/time/time-zones-interesting.html).